# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/mpc/hotfix-stdio.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./src/mpc.h.vanilla	2022-12-12 17:30:44.077432387 +0100
+++ ./src/mpc.h	2022-12-12 17:43:50.771465109 +0100
@@ -284,7 +285,9 @@
    int64_t exp);
 __MPC_DECLSPEC void mpcr_max (mpcr_ptr r, mpcr_srcptr s, mpcr_srcptr t);
 __MPC_DECLSPEC int64_t mpcr_get_exp (mpcr_srcptr r);
+#ifdef _GMP_H_HAVE_FILE
 __MPC_DECLSPEC void mpcr_out_str (FILE *f, mpcr_srcptr r);
+#endif
 __MPC_DECLSPEC void mpcr_mul (mpcr_ptr r, mpcr_srcptr s, mpcr_srcptr t);
 __MPC_DECLSPEC void mpcr_mul_2ui (mpcr_ptr r, mpcr_srcptr s,
    unsigned long int e);
